packages/freshmint/flow/index.ts

```diff
@@ -230,6 +230,12 @@ export class FlowGateway {
 
     return parseGetEditionResults(results);
   }
+
+  async destroyNFTs(ids: string[]) {
+    return await this.cli.transaction('./cadence/transactions/destroy_nfts.cdc', this.signer, [
+      { type: t.Array(t.String), value: ids },
```

Comment:
```suggestion
      { type: t.Array(t.String), value: ids },
      { type: t.Optional(t.String), null } // bucketName
```

You'll need to add the `fromBucketName` argument here since the transaction expects it. The CLI doesn't have support for buckets yet, so it can just be `null` for now.